### PR TITLE
[ALLUXIO-2108] Add support to publish AlluxioMaster WebUI url to mesos UI under alluxio framework

### PR DESCRIPTION
**Pull request by shaileshcheke**
https://alluxio.atlassian.net/browse/ALLUXIO-2108

### CODE REVIEW

File: integration/mesos/src/main/java/alluxio/mesos/AlluxioFramework.java [hunk 1]
```diff
@@ -72,6 +72,14 @@ public void run() {
       frameworkInfo.setPrincipal(Configuration.get(PropertyKey.INTEGRATION_MESOS_PRINCIPAL));
     }
 
+    // Publish WebUI url to mesos master.
+    String masterHostName = Configuration.containsKey(PropertyKey.MASTER_WEB_HOSTNAME)
```
Comment (aaudiber) [inline integration/mesos/src/main/java/alluxio/mesos/AlluxioFramework.java]: We have a utility get the master connect address from the configuration:
```java
NetworkAddressUtils.getConnectAddress(ServiceType.MASTER_WEB).toString()
```

Comment (shaileshcheke) [reply integration/mesos/src/main/java/alluxio/mesos/AlluxioFramework.java]: Thanks @aaudiber.

**Comment by shaileshcheke** [comment]
@aaudiber I made changes .Please review.Thanks.

**Comment by shaileshcheke** [comment]
Thanks @yupeng9 and @aaudiber.

**Pull request merged**